package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "skyux-tree-view",
+  "version": "0.0.1",
+  "description": "SKYUX styles and utilities to be used with angular-tree-view",
+  "main": "bundles/bundle.umd.js",
+  "module": "index.ts",
+  "scripts": {
+    "build": "skyux build-public-library",
+    "test": "skyux test --coverage library",
+    "watch": "skyux watch --coverage library",
+    "start": "skyux serve",
+    "e2e": "skyux e2e",
+    "pact": "skyux pact",
+    "pact:watch": "skyux pact --watch",
+    "lint": "skyux lint"
+  },
+  "keywords": [],
+  "author": "Blackbaud",
+  "license": "ISC",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/blackbaud/skyux-tree-view"
+  },
+  "peerDependencies": {
+    "@angular/common": "^7.0.0",
+    "@angular/core": "^7.0.0",
+    "@skyux/i18n": "^3.7.0",
+    "@skyux/indicators": "^3.0.0",
+    "@skyux/layout": "^3.2.0",
+    "angular-tree-component": "^8.4.0"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@angular/animations": "7.2.15",
+    "@angular/common": "7.2.15",
+    "@angular/compiler": "7.2.15",
+    "@angular/compiler-cli": "7.2.15",
+    "@angular/core": "7.2.15",
+    "@angular/forms": "7.2.15",
+    "@angular/http": "7.2.15",
+    "@angular/platform-browser": "7.2.15",
+    "@angular/platform-browser-dynamic": "7.2.15",
+    "@angular/router": "7.2.15",
+    "@blackbaud/auth-client": "2.17.1",
+    "@pact-foundation/pact-web": "7.4.0",
+    "@skyux-sdk/builder": "3.8.0",
```

Comment:
Some of these packages are a few versions behind; might be worth running `skyux upgrade` and then `npm audit` to fix any vulnerability warnings.

Reply:
Done.